README.md

```diff
@@ -261,24 +267,54 @@ and vice-versa. We have created this [codesandbox](https://codesandbox.io/s/213m
 
 Let's add some children to the AppBar to make it more realistic.
 
-Let import `Button`, `Heading`, and `Notification` icon.
+Let import `Button`, `Header`, `Heading`, and `Notification` icon.
 
 ```diff
 - import { Box, Grommet } from 'grommet';
-+ import { Box, Button, Heading, Grommet } from 'grommet';
++ import { Box, Button, Header, Heading, Grommet } from 'grommet';
 + import { Notification } from 'grommet-icons';
 ```
 
+Now we can swap out the Box in the AppBar for the Header. Header is a specialized version of the Box component that defines some of the props commonly used for a header component automatically.
+
+```diff
+const AppBar = (props) => (
+-  <Box
+-    tag="header"
+-    direction="row"
+-    align="center"
+-    justify="between"
+-    background="brand"
+-    pad={{ left: "medium", right: "small", vertical: "small" }}
+-    elevation="medium"
+-    {...props}
+-  />
++ <Header
++   background="brand"
++   pad={{ left: "medium", right: "small", vertical: "small" }}
++   elevation="medium"
++   {...props}
++ />
+);
+```
+
 Update AppBar children to the following:
 
 ```diff
   <AppBar>
 -    Hello Grommet!
-+   <Heading level='3' margin='none'>My App</Heading>
++   <Heading level='1' margin='none'>My App</Heading>
 +   <Button icon={<Notification />} onClick={() => {}} />
   </AppBar>
 ```
 
+Notice that the 'My App' Heading feels a bit large. We can adjust the font size of the Heading with the size prop. Changing the level will also automatically change the font size. However, it is very important that headings use the [correct level](https://www.w3.org/WAI/tutorials/page-structure/headings/) for accessibility.
+
+```diff
+-   <Heading level='1' margin='none'>My App</Heading>
++   <Heading level='1' size="small" margin='none'>My App</Heading>
+```
+
 ## Adding a main body
```

Comment:
Should we introduce `page` and `pagecontent` here or do that separate ?

Reply:
I adjusted the app so that it demonstrates using Page, PageContent, and PageHeader. I also removed the sidebar since I don't think we want to encourage this pattern.